### PR TITLE
Fixed most typos

### PR DESCRIPTION
**Pull request by jonaswouters**
I might have missed some. 🤷‍♂

### CODE REVIEW

File: topics/go/packages/encoding/example1/example1.go [hunk 1]
```diff
@@ -38,7 +38,7 @@ type (
 		CurrencyCode string `json:"currency_code"`
 		TimeZone     string `json:"time_zone"`
 		NumberFormat struct {
-			DecimalSaparator  string `json:"decimal_separator"`
+			DecimalSeparator  string `json:"decimal_separator"`
```
Comment (ardan-bkennedy) [inline topics/go/packages/encoding/example1/example1.go]: We need to update the README with a new playground link for this change.

File: topics/go/packages/context/exercises/template1/template1.go [hunk 2]
```diff
@@ -32,7 +32,7 @@ func routes() {
 // both timeouts and state.
 func findUser(rw http.ResponseWriter, r *http.Request) {
 
-	// Create a context that timesout in fifty milliseconds.
+	// Create a context that timeouts in fifty milliseconds.
```
Comment (ardan-bkennedy) [inline topics/go/packages/context/exercises/template1/template1.go]: We need to update the README with a new playground link for this change.

File: topics/go/packages/context/exercises/exercise1/exercise1.go [hunk 3]
```diff
@@ -43,7 +43,7 @@ func routes() {
 // findUser makes a database call to find a user.
 func findUser(rw http.ResponseWriter, r *http.Request) {
 
-	// Create a context that timesout in fifty milliseconds.
+	// Create a context that timeouts in fifty milliseconds.
```
Comment (ardan-bkennedy) [inline topics/go/packages/context/exercises/exercise1/exercise1.go]: We need to update the README with a new playground link for this change.

File: topics/go/packages/context/README.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ## Context - Standard Library
 
-The package context defines the Context type, which carries deadlines, cancelation signals, and other request-scoped values across API boundaries and between processes.
+The package context defines the Context type, which carries deadlines, cancellation signals, and other request-scoped values across API boundaries and between processes.
```
Comment (ardan-bkennedy) [inline topics/go/packages/context/README.md]: We need to update the README with a new playground link for this change.

File: topics/go/language/methods/example1/example1.go [hunk 5]
```diff
@@ -54,7 +54,7 @@ func main() {
 	}
 
 	// Exception example: Using pointer semantics
-	// for a collectoin of strings.
+	// for a collection of strings.
```
Comment (ardan-bkennedy) [inline topics/go/language/methods/example1/example1.go]: We need to update the README with a new playground link for this change.

**Comment by ardan-bkennedy** [comment]
This is brilliant!!  A handful or more of these changes have a go playground link that needs to be updated
:(

Do you think you can do that as well?

File: topics/go/language/embedding/exercises/exercise1/exercise1.go [hunk 6]
```diff
@@ -48,7 +48,7 @@ type CachingFeed struct {
 	*Feed
 }
 
-// NewCachingFeed initalizes a CachingFeed for use.
+// NewCachingFeed initializes a CachingFeed for use.
```
Comment (ardan-bkennedy) [inline topics/go/language/embedding/exercises/exercise1/exercise1.go]: We need to update the README with a new playground link for this change.

File: topics/go/language/embedding/exercises/template1/template1.go [hunk 7]
```diff
@@ -74,7 +74,7 @@ type CachingFeed struct {
 	// TODO embed *Feed and add a field for a map[string]Document.
 }
 
-// NewCachingFeed initalizes a CachingFeed for use.
+// NewCachingFeed initializes a CachingFeed for use.
```
Comment (ardan-bkennedy) [inline topics/go/language/embedding/exercises/template1/template1.go]: We need to update the README with a new playground link for this change.

File: topics/go/packages/reflection/exercises/exercise1/exercise1.go [hunk 8]
```diff
@@ -3,7 +3,7 @@
 
 // Declare a struct type that represents a request for a customer invoice. Include a CustomerID and InvoiceID field. Define
 // tags that can be used to validate the request. Define tags that specify both the length and range for the ID to be valid.
-// Declare a function named validate that accepts values of any type and processes the tags. Display the resutls of the validation.
+// Declare a function named validate that accepts values of any type and processes the tags. Display the results of the validation.
```
Comment (ardan-bkennedy) [inline topics/go/packages/reflection/exercises/exercise1/exercise1.go]: We need to update the README with a new playground link for this change.

File: topics/go/packages/reflection/exercises/template1/template1.go [hunk 9]
```diff
@@ -3,7 +3,7 @@
 
 // Declare a struct type that represents a request for a customer invoice. Include a CustomerID and InvoiceID field. Define
 // tags that can be used to validate the request. Define tags that specify both the length and range for the ID to be valid.
-// Declare a function named validate that accepts values of any type and processes the tags. Display the resutls of the validation.
+// Declare a function named validate that accepts values of any type and processes the tags. Display the results of the validation.
```
Comment (ardan-bkennedy) [inline topics/go/packages/reflection/exercises/template1/template1.go]: We need to update the README with a new playground link for this change.

Comment (ardan-bkennedy) [reply topics/go/packages/reflection/exercises/template1/template1.go]: I've tried to identify the best I can but I could be missing something.

**Comment by jonaswouters** [comment]
That's a lot of playground links that need updating. I'll have a look at it soon.

**Comment by jonaswouters** [comment]
Updated the playground links. There was 1 playground link that was completely different than the sourcefile.

**Comment by ardan-bkennedy** [comment]
This was very nice of you! I don't have time to check all the playground links so I will trust them. 

THANK YOU

**Pull request merged**